astrokat/test/test_offline_observe.py

```diff
@@ -1,111 +1,93 @@
+from __future__ import absolute_import
 from __future__ import print_function
+
+
 import unittest2 as unittest
-import sys
+from mock import patch
 
-from astrokat import observe_main, simulate
 
-import logging
-if sys.version_info[0] == 3:
-    from io import StringIO
-else:
-    from StringIO import StringIO
+from .testutils import LoggedTelescope, execute_observe_main
 
+from six.moves import StringIO
 
-class test_astrokat_yaml(unittest.TestCase):
 
-    def setUp(self):
-        user_logger = logging.getLogger('astrokat.simulate')
+@patch('astrokat.observe_main.Telescope', LoggedTelescope)
+class TestAstrokatYAML(unittest.TestCase):
 
-        # remove current handlers
-        for handler in user_logger.handlers:
-            user_logger.removeHandler(handler)
+    def setUp(self):
+        LoggedTelescope.user_logger_stream = StringIO()
```

Comment:
Is this to reset the stream after each test?  If yes, suggest a comment saying something like that.  Or better yet, add a static method with a friendly name to `LoggedTelescope` that performs this action.

Also, suggest we add the `setUp` method to `astrokat/test/test_nd_timings.py` so that it doesn't break if a second test is ever added.